events/moose/moose.go

```diff
@@ -365,7 +364,7 @@ func (s *Subscriber) NotifyConnect(data events.DataConnect) error {
 	}
 	return s.response(moose.Send_serviceQuality_servers_connect(
 		-1,
-		dnsResolutionTime,
+		int(time.Since(s.connectedAt).Seconds()),
```

Comment:
This will still not be correct as this function is called with `EventStatusAttempt`  after potential API request is already done

Reply:
The `connecting` event is sent before picking up the VPN server.